java/src/main/java/io/vertx/eventbusclient/transport/WebSocketTransport.java

```diff
@@ -76,8 +76,12 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
         reading = true;
         if (msg instanceof BinaryWebSocketFrame) {
           BinaryWebSocketFrame frame = (BinaryWebSocketFrame) msg;
-          String json = frame.content().toString(StandardCharsets.UTF_8);
-          messageHandler.handle(json);
+          try {  //TODO: @wjw_note: release BinaryWebSocketFrame Prevent resource leakage!
```

Comment:
What is the purpose of this line ? can we remove it ? others LGTM

Reply:
use `try finally` promise release BinaryWebSocketFrame Prevent resource leakage!

Reply:
sorry, I meant the comment: `//TODO: @wjw_note: release BinaryWebSocketFrame Prevent resource leakage!
`

Reply:
This is the note I added

Reply:
Sorry again that I did not make myself clear, and to be precise, it is `TODO: @wjw_note:`, I don't want to be captious, but `TODO` means something not ready, and the `@wjw_note` looks like not this component related.

Reply:
Sorry, the `TODO & wjw__note` in this comment is misleading